WordPress/create-block-theme
# Update package-lock.json

I noticed that when I run `npm ci` on the latest trunk, `package-lock.json` gets updated. I think this is because only `@wordpress/env` in `package.json` was updated in #619.

### Testing Instructions

- Check out this branch and run `npm i`.
- Run `git diff`
- There should be no difference.
